proto/provenance/metadata/v1/tx.proto

```diff
@@ -122,6 +127,52 @@ message MsgDeleteScopeRequest {
 // MsgDeleteScopeResponse is the response type for the Msg/DeleteScope RPC method.
 message MsgDeleteScopeResponse {}
 
+// MsgAddScopeDataAccessRequest is the request to add data access metaaddresses to scope
+message MsgAddScopeDataAccessRequest {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer)         = false;
+  option (gogoproto.goproto_getters)  = false;
+
+  // scope MetadataAddress for updating data access
+  bytes scope_id = 1 [
+    (gogoproto.nullable)   = false,
+    (gogoproto.customtype) = "MetadataAddress",
+    (gogoproto.moretags)   = "yaml:\"scope_id\""
+  ];
+
+  // bech32 address to be added to scope
+  repeated string data_access = 2 [(gogoproto.moretags) = "yaml:\"data_access\""];
+  // signers is the list of address of those signing this request.
+  repeated string signers = 3;
+}
+
+// MsgAddScopeDataAccessResponse is the response for adding data access metaaddresses to scope
+message MsgAddScopeDataAccessResponse {}
+
+// MsgDeleteScopeDataAccessRequest is the request to remove data access metaaddresses to scope
+message MsgDeleteScopeDataAccessRequest {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer)         = false;
+  option (gogoproto.goproto_getters)  = false;
+
+  // scope MetadataAddress for removing data access
+  bytes scope_id = 1 [
+    (gogoproto.nullable)   = false,
+    (gogoproto.customtype) = "MetadataAddress",
+    (gogoproto.moretags)   = "yaml:\"scope_id\""
+  ];
+
+  // bech32 address to be removed from scope
```

Comment:
Not totally sold on the "bech32 addresses" title here... the next line just uses "address" to describe these values... if we were being more formal the term is "AccAddress"